content/applications/marketing/events/event_booths.rst

```diff
@@ -0,0 +1,242 @@
+that checkbox is ticked, that means that when a booth belonging to this booth category is booked,
+the user is created as an official *Sponsor* of the event.
+
+Also, when the :guilabel:`Sponsorship` checkbox is ticked, two additional fields appear beneath it:
+:guilabel:`Sponsor Level` and :guilabel:`Sponsor Type`.
+
+Select a desired level of sponsorship from the :guilabel:`Sponsor Level` drop-down field.
+
+.. tip::
+   To modify any existing :guilabel:`Sponsor Level`, select it from the drop-down field, then click
+   the :guilabel:`➡️ (right arrow)` that appears at the end of the line. Doing so a separate page,
+   wherein the :guilabel:`Sponsor Level` name and :guilabel:`Ribbon Style` can be changed, if
+   necessary.
+
+Users can also create a new :guilabel:`Sponsor Level`, by typing in the name of the new level, and
+clicking :guilabel:`Create and edit...` from the resulting drop-down menu.
+
+Doing so reveals a :guilabel:`Create Sponsor Level` pop-up window.
+
+.. image:: event_booths/create-sponsor-level-popup.png
+   :align: center
+   :alt: The Create Sponsor Level pop-up window that appears in the Odoo Events application.
+
+From this pop-up window, confirm the newly-created :guilabel:`Sponsor Level`, and decide what kind
+(if any) :guilabel:`Ribbon Style` should be applied. The :guilabel:`Ribbon Style` options available
+in that drop-down field are: :guilabel:`No Ribbon`, :guilabel:`Gold`, :guilabel:`Silver`, and
+:guilabel:`Bronze`.
+
+That :guilabel:`Ribbon Style` (if one is selected) appears with the sponsor's name on the event
+website.
+
+.. note::
+   Clicking :guilabel:`Create` from the resulting drop-down menu in this instance creates the
+   sponsor level, but doesn't immediately prompt the user to further configure it, via a
+   :guilabel:`Create Sponsor Level` pop-up window.
+
+Beneath those sections (:guilabel:`Booth Details` and :guilabel:`Sponsorship`), there is the
+:guilabel:`Description` tab. In this tab, proceed to enter any vital information related to the
+booth category that would be important for any potential booth-buyer to know about (e.g. the square
+footage, any amenities, size of display screen, etc.).
+
+Add booth to an event
+=====================
+
+In order to add a booth to an event, navigate to an existing event form, via :menuselection:`Events
+app --> Events`, and select the desired event from the :guilabel:`Events` dashboard. Or, click
+:guilabel:`New` to open a blank event form.
+
+From the event form, to access the *Booths* for that specific event, click the :guilabel:`Booths`
+smart button at the top of the page.
+
+The :guilabel:`Booths` page is displayed in a Kanban view, by default, with two different stages:
+:guilabel:`Available` and :guilabel:`Unavailable`.
+
+.. note::
+   The :guilabel:`Booths` page of an event is also viewable in a :icon:`oi-view-list`
+   :guilabel:`List` view, :icon:`fa-area-chart` :guilabel:`Graph` view, and :icon:`oi-view-pivot`
```

Comment:
Question about the graph view - is it represented by this Odoo UI icon? : 

oi-view-cohort

I think it might be, if it is I'd say to use that one

Reply:
Actually, oi-view-cohort is for the cohort view. the FA icon I have here represents the appropriate icon, so I'm going to stick with that, unless SALI instructs me to change it.